docs/source/user/aerodyn/input.rst

```diff
@@ -207,13 +214,13 @@ The input parameters in this section are used only when ``AFAeroMod
 
 ``UAMod`` determines the UA model. It has the following options:
 
-- ``1``: the original theoretical developments of B-L (**not currently functional**), 
-- ``2``: the extensions to B-L developed by González 
-- ``3``: the extensions to B-L developed by Minnema/Pierce
-- ``4``: a continuous-state model developed by Hansen, Gaunna, and Madsen (HGM)
-- ``5``: a model similar to HGM with an additional state for vortex generation
-- ``6``: Oye's dynamic stall model
-- ``7``: Boeing-Vertol model
+- ``1``: the discrete-time model of Beddoes-Leishman (B-L) (**not currently functional**), 
+- ``2``: the extensions to B-L developed by González  (changes in Cn, Cc, Cm)
+- ``3``: the extensions to B-L developed by Minnema/Pierce (changes in Cc and Cm)
+- ``4``: 4-states continuous-time B-L model developed by Hansen, Gaunna, and Madsen (HGM). NOTE: might require smaller time steps until a stiff integrator is implemented.
+- ``5``: 5-states continuous-time B-L model similar to HGM with an additional state for vortex generation
+- ``6``: 1-state continuous-time developed by Oye
+- ``7``: discrete-time Boeing-Vertol (BV) model
 
 The models are described in :numref:`AD_UA`.
```

Comment:
Suggest adding here something like, "UAMod = 4,5,6 all support linearization, but 1,2,3, and 7 do not."